testsuites/CBLTester/CBL_Functional_tests/APITests_sc/test_basic_sc.py

```diff
@@ -0,0 +1,44 @@
+import random
+import pytest
+from keywords.utils import random_string
+
+
+@pytest.mark.usefixtures("class_init")
+class TestBasicSC(object):
+
+    def test_create_scope(self, scope_name):
+        ''' Create scope with different name '''
+        self.server_obj.create_scope(scope_name)
+
+    def test_create_collection(self, scope_name, collection_name):
+        '''
+        Test create collection
+        '''
+        self.server_obj.create_collection()
+
+    def test_create_scope_already_exist(self, scope_name):
```

Comment:
I wonder if we need that test - maybe the developers have the same test running in their suite?

Reply:
This test to ensure that data in current scope or collection will not be deleted if we create a new ones with same name

Reply:
Do the developers have that test in their test suite? If they do, maybe it's not worth adding it, because it takes time to run with the whole setup, etc.?